.github/workflows/pipeline.yml

```diff
@@ -41,6 +43,10 @@ jobs:
           | sed 's/\\ $//' > release_notes.md
           cat release_notes.md
 
+      - name: Get RBAC version
+        id: get_rbac_version
+        run: awk -F'=' '/^${{ github.event.head_commit.message }}=/ { print "rbac_version=" $2 }' rbac-versions.txt >> $GITHUB_OUTPUT
```

Comment:
What happens if it doesn't find a matching line?  Could that cause the `${{ needs.release.outputs.rbac_version }}` reference to fail further down the workflow?

Reply:
Just tested this on a local cut-down repo. Since the awk command is printing the `rbac_version=` bit every time, it'll just set the var to an empty string. The empty string gets passed into the deploy.yml file and then gets caught by the if check in the deploy job.

Reply:
Nice, thanks!